interfaces/apparmor/template.go

```diff
@@ -203,6 +203,7 @@ var templateCommon = `
   /run/systemd/journal{,.snap-*}/socket w,
   /run/systemd/journal{,.snap-*}/stdout rw, # 'r' shouldn't be needed, but journald
                                             # doesn't leak anything so allow
+  /run/systemd/journal{,.snap-*}/dev-log w,
```

Comment:
Consolidate these to simplify profile generation
```suggestion
  # 'r' shouldn't be needed, but journald doesn't leak anything so allow
  /run/systemd/journal{,.snap-*}/{dev-log, stdout} rw,
                                            
```

Reply:
I think I'd keep it separate for better visibility. If there's any optimization to be done, it's likely already part of the relevant passes in the parser.

Reply:
Also this then grants `r` permission to `dev-log` which apparently is not strictly needed, so if anything it would be best to amalgamate `socket` and `dev-log` which both have just `w` permission.

Reply:
I haven't find any information about the optimization/amalgamation of apparmor profiles. 
If this doesn't affect the performance, I tend to keep it separate. So, should I put `dev-log` and `socket` together?

Reply:
No that is fine - keep it separate for now until we have a good reason to think it needs to be optimised manually.